docs/Animations.md

```diff
@@ -444,14 +445,14 @@ var App = React.createClass({
 
     return (
       <View style={styles.container}>
-        <TouchableWithoutFeedback onPressIn={this._onPressIn}
-                                  onPressOut={this._onPressOut}>
+        <TouchableWithoutFeedback onPressIn={() => this._onPressIn()}
+                                  onPressOut={() => this._onPressOut()}>
```

Comment:
@sunnylqm would you mind following @satya164’s suggestion and bind in the constructor? I’d shipit